pypolsys/utils.py

```diff
@@ -22,29 +22,31 @@
 partitions to `POLSYS_PLP`.
 
 Changing the partition has a strong impact on the number of paths to track
-in the homotopy. `PPLSYS_PLP` supports
+in the homotopy. `POLSYS_PLP` supports
   * 1-homogeneous, denoted by h,
   * m-homogeneous, denoted by mh,
   * partitioned linear product, denoted by plp.
 
-For more details on `PPLSYS_PLP` partition, See
+For more details on `POLSYS_PLP` partition, See
 > Wise, Steven M., Andrew J. Sommese, and Layne T. Watson. "Algorithm 801:
 > POLSYS_PLP: A partitioned linear product homotopy code for solving polynomial
 > systems of equations." ACM Transactions on Mathematical Software (TOMS)
 > 26.1 (2000): 176-200.
 
 For discussion on partitions and polynomials system types (dense, sparse, determinental), see
 > Verschelde, Jan. "Polynomial homotopies for dense, sparse and determinantal
-> systems." arXiv preprint math/9907060 (1999).
+> systems", _arXiv preprint math/9907060_ (1999).
 and the references therein.
 """
 
 import numpy as np
 from pypolsys import polsys
 
+
 def fromSympy(P):
-    """ Create polsys polynomial from sympy list of Poly. All variables
-    should be present in the generator.
+    """Create polsys polynomial from sympy list of Poly.
+
+    All variables should be present in the generator.
```

Comment:
"present" ? Wondering if "instantiated" would not be better ?

Reply:
Ok I will fix it.

Reply:
Thx ;-)